tst/testinstall/varargs.tst

```diff
@@ -74,22 +74,10 @@ gap> function(a,b....) end;
 Syntax error: ) expected in stream:1
 function(a,b....) end;
                ^
-gap> f := function(a,b..) end;
-Syntax error: Three dots required for variadic argument list in stream:1
-f := function(a,b..) end;
-                 ^^
 gap> Display(RETURN_FIRST);
 function ( first, rest... )
     <<kernel code>> from src/gap.c:RETURN_FIRST
 end
-gap> Print(INSTALL_METHOD_FLAGS,"\n");
-function ( opr, info, rel, flags, baserank, method )
-    <<compiled GAP code>> from GAPROOT/lib/oper1.g:147
-end
```

Comment:
This test actually didn't even make sense in this file, so I am not sure why it was here to begin with